.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+image:
+  file: .gitpod.Dockerfile
+vscode:
+  extensions:
+    - ms-python.python
+tasks:
+  - name: install-dependencies 
+    init: |
+      pip install -r requirements.txt
```

Comment:
@kylos101 Why the two separate yml files? Would it not be more effective to have .github/workflows/pylint.yml or .gitpod.yml govern initialization behavior globally rather than sequester yml files in this way?